neofs-testlib/neofs_testlib/env/env.py

```diff
@@ -195,16 +195,52 @@ def generate_wallet(
 
         with open(prepared_wallet.path, "w") as wallet_file:
             json.dump(wallet_json, wallet_file)
-        ###
 
         prepared_wallet.address = wallet_utils.get_last_address_from_wallet(
             prepared_wallet.path, prepared_wallet.password
         )
 
+    @allure.step("Generate alphabet wallets")
+    def generate_alphabet_wallets(
+        self,
+        network_config: Optional[str] = None,
+        size: Optional[int] = 1,
+    ) -> list[NodeWallet]:
+        neofs_adm = self.neofs_adm(network_config)
+
+        neofs_adm.morph.generate_alphabet(alphabet_wallets=self.alphabet_wallets_dir, size=size)
+
+        generated_wallets = []
+
+        for generated_wallet in os.listdir(self.alphabet_wallets_dir):
+            # neo-go requires some attributes to be set
```

Comment:
I don't think so.

Reply:
it fails without these attributes:

```
E           jsonschema.exceptions.ValidationError: 'name' is a required property
```

it comes from 

```
.venv/lib/python3.12/site-packages/neo3/wallet/wallet.py:304: in from_json
    validate(json, schema=cls.json_schema)
```

Reply:
But it's not a neo-go problem, isn't it?

Reply:
updated the comment